internal/server/deviceflow.go

```diff
@@ -91,6 +95,7 @@ func (a *API) GetDeviceFlowStatus(c *gin.Context, req *api.PollDeviceFlowRequest
 				Name:      dfar.AccessKey.IssuedForName,
 				AccessKey: dfar.AccessKeyToken,
 				Expires:   api.Time(dfar.AccessKey.ExpiresAt),
+				// TODO: set OrganizationName for consistency with other login methods
```

Comment:
Reminder to myself to resolve this TODO (unless someone knows of a reason we shouldn't).

Reply:
I started writing a test for this and noticed the user name in the response is also missing. So I'm going to fix these in a separate PR.